docs/concepts/typings.md

```diff
@@ -4,4 +4,103 @@ title: Type hints
 
 # Type hints
 
-...
+Type hints are a modern feature of Python available since 3.5 whose existence is heavily influenced by the features of type-safe languages such as Rust. To learn more about type hints in Python, see [Real Python’s Type hinting walkthrough](https://realpython.com/lessons/type-hinting/).
+
+When using Strawberry to build graphQL APIs, as was shown in [Schema basics](https://strawberry.rocks/docs/general/schema-basics), type hints are required within classes decorated by `@strawberry.type` & `@strawberry.input` and functions decorated by `@strawberry.field` & `strawberry.mutation`. These type hints are sourced as the keywords `str`, `int`, `float`, and from packages imported directly from the Python standard libraries `typing`, that has been available since Python 3.5, `datetime`, and `decimal`.
+
+## Mapping to graphQL types
+
+The complete mapping of the required type hints for the relevant graphQL types is as follows:
+
+| GraphQL       | Python                     |
+| ------------- | -------------------------- | --- |
+| `ID`          | `strawberry.ID`            |
+| `String`      | `str`                      |
+| `Integer`     | `int`                      |
+| `Float`       | `float`                    |
+| `Decimal`     | `decimal.Decimal`          |
+| `Array`, `[]` | `typing.List` or `list`    |
+| `Union`       | `typing.Union` or `        | `   |
+| `Nullable`    | `typing.Optional` or `None | `   |
```

Comment:
Is `Nullable` a real GraphQL type? I don't think it should be in backticks here

Reply:
It isn't a type by its own right but it is an important GraphQL concept. The translation of ! from the SDL to Strawberry resolver definitions is important, especially as a code-first library.
I included it here because it was something that I tripped over when I first came to the library. The logic is reversed in Graphene; fields are null by default with `required=True` needed to make it non-null.